src/OpenLoco/OpenLoco.cpp

```diff
@@ -375,7 +350,7 @@ namespace OpenLoco
     {
         if (isAlreadyRunning("Locomotion"))
         {
-            exitWithError(StringIds::game_init_failure, StringIds::loco_already_running);
+            // exitWithError(StringIds::game_init_failure, StringIds::loco_already_running);
```

Comment:
did you mean to delete this?

Reply:
Yes and no, it is very annoying to develop and test multiplayer when you can't launch multiple instances. I would like to just delete this restriction.